### PR TITLE
feat:buttons added

### PR DESCRIPTION
**Pull request by nyyshaaa**
btn1

### CODE REVIEW

File: pages/components/Button/button.css [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+}
+
+.btn{
+    border: none;
+    background: none;
+    padding: none;
+    font-size: inherit;
+}
+
+.btn-primary{
```
Comment (khyatibaria) [inline pages/components/Button/button.css:15]: suggestion can add `cursor: pointer;` property

**Review by khyatibaria — COMMENTED** [review]
looks good to merge just added one comment check.

File: pages/components/Button/button.css [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    vertical-align: middle;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    background-color: var(--white);
+    color: var(--blue-purple);
+    border: 1px solid var(--white);
+   
+
+}
```
Comment (Rockyraven) [inline pages/components/Button/button.css:55]: remove this extra space

File: pages/components/Button/button.css [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+    top: 0;
+    left: 0;
+    position: absolute;
+    background-image: linear-gradient(to right,var(--sky-blue) 0%,var(--shine-green) 100%);
+    display: block;
+    transition: .5s ease;
+    z-index: -1;
+   
+
+}
```
Comment (Rockyraven) [inline pages/components/Button/button.css:71]: here also

File: pages/components/Button/button.css [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+.btn-grad span{
+    color: var(--sky-blue);
+    padding-right: 0.2rem;
+}
+
+
+.link-cta{
+    display: inline-block;
+    /* text-align: center; */
+    width: 14rem;
```
Comment (Rockyraven) [inline pages/components/Button/button.css:90]: remove that wanted code and extra space

File: pages/components/Button/button.css (hunk 2 above)
Comment (nyyshaaa) [reply pages/components/Button/button.css:55]: yeah will do in end , i just forgot,thanks!

File: pages/components/Button/button.css (hunk 1 above)
Comment (nyyshaaa) [reply pages/components/Button/button.css:15]: i forgot to add it ,thank you!

File: pages/components/Button/button.css (hunk 4 above)
Comment (nyyshaaa) [reply pages/components/Button/button.css:90]: okeiy

**Comment by nyyshaaa** [comment]
feat:button component done

**Pull request merged**